fili-core/src/main/java/com/yahoo/bard/webservice/async/jobs/stores/ApiJobStoreFilterOperation.java

```diff
@@ -0,0 +1,10 @@
+// Copyright 2016 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE file distributed with this work for terms.
+package com.yahoo.bard.webservice.async.jobs.stores;
+
+/**
+ * Types of legal ApiJobStoreFilter operations.
+ */
+public enum ApiJobStoreFilterOperation {
+    eq
```

Comment:
I initially used FilterOperation but since we are currently only supporting eq, I created a special one. Yea yea I know we can throw IllegalArgumentException. I will correct this